scripts/jenkins/ardana/ansible/roles/ardana_deploy/defaults/main.yml

```diff
@@ -15,6 +15,8 @@
 #
 ---
 
+ardana_nova_migrate_enabled: True
```

Comment:
So my concerns would be with adding this as enabled by default... So long as it doesn't lead to side effects causing an increase in CI/Gating test instability it should be fine, especially if the associated migration tests aren't actually enabled.

Reply:
agreed, I tried it before and did not notice any increase on instability. I we start seeing issues with this, we can default it to false.